### PR TITLE
chore: add Imran Khan

### PR DESCRIPTION
**Pull request by imran201khan**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/838"></a>

### CODE REVIEW

File: public/data/imran.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Imran Khan",
+    "type": "personal",
+    "bio": "Open Source Enthusiast !",
+    "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
```
Comment (Panquesito7) [inline public/data/imran.json:5]:
```suggestion
    "avatar": "https://github.com/imran201khan.png",
```

File: public/data/imran.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Imran Khan",
+    "type": "personal",
+    "bio": "Open Source Enthusiast !",
```
Comment (Panquesito7) [inline public/data/imran.json:4]: Would you like to add a bit more of yourself? 🙂

File: public/data/imran.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "milestones": [
+      {
+        "title": "DSA",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing",
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/imran.json:27]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/imran.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "icon": "Linkden"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "DSA",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing",
```
Comment (krishguptadev) [inline public/data/imran.json:24]:
```suggestion
        "description": "Started freelancing"
```

File: public/data/imran.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/imran201khan",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/imrankhan",
+        "icon": "Linkden"
```
Comment (syedsohan) [inline public/data/imran.json:15]: "icon": "twitter"

Comment (syedsohan) [reply public/data/imran.json:15]: As it is a Twitter profile link so change the icon LinkedIn to Twitter.

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/imran.json (hunk 5 above)
Comment (syedsohan) [inline public/data/imran.json:15]: As it is your Twitter profile so change the icon from LinkedIn to Twitter.

```suggestion
        "icon": "twitter"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/imran.json (hunk 4 above)
Comment (abdussalam02) [reply public/data/imran.json:24]: I think the description should match the title..

Comment (krishguptadev) [reply public/data/imran.json:24]: I've not changed the description, just removed the trailing comma.

**Comment by emmalearnscode** [comment]
@imran201khan Do you need any help making the requested changes in your pull request?

**Review by Jassi10000 — COMMENTED** [review]
@imran201khan Is there any update on your PR buddy after repo maintainers made their suggestions?

**Comment by KendallDoesCoding** [comment]
@imran201khan Hasnt made any commit on his github for the last 18 days

@emmalearnscode @Panquesito7 Feel free to close this for now if @imran201khan isnt answering.

**Comment by emmalearnscode** [comment]
@imran201khan I’ll close this for now but feel free to open a new pull request or reopen this one if you get time to come back and fix this.

**Comment by KendallDoesCoding** [comment]
> @imran201khan I’ll close this for now but feel free to open a new pull request or reopen this one if you get time to come back and fix this.

If a maintainer or a mod closes the issue, the original user cannot reopen the PR, so @imran201khan if you want to come back to this PR and fix it, please let the maintainer or mods know. Thanks

**Pull request closed without merging**